<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="21560" started-at="2019-07-14T13:23:00Z" finished-at="2019-07-14T13:23:22Z">
    <groups>
    </groups>
    <test name="newCustomer" duration-ms="21560" started-at="2019-07-14T13:23:00Z" finished-at="2019-07-14T13:23:22Z">
      <class name="NotPartOfFramework.LoginListenersTests_UsingSuiteFile">
        <test-method status="PASS" signature="readConfiguration()[pri:0, instance:NotPartOfFramework.LoginListenersTests_UsingSuiteFile@64bf3bbf]" name="readConfiguration" is-config="true" duration-ms="3527" started-at="2019-07-14T18:52:57Z" finished-at="2019-07-14T18:53:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readConfiguration -->
        <test-method status="PASS" signature="beforeMethod(org.testng.ITestContext)[pri:0, instance:NotPartOfFramework.LoginListenersTests_UsingSuiteFile@64bf3bbf]" name="beforeMethod" is-config="true" duration-ms="14427" started-at="2019-07-14T18:53:00Z" finished-at="2019-07-14T18:53:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1698ee84]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="Login_TC1()[pri:0, instance:NotPartOfFramework.LoginListenersTests_UsingSuiteFile@64bf3bbf]" name="Login_TC1" duration-ms="3269" started-at="2019-07-14T18:53:15Z" finished-at="2019-07-14T18:53:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at org.testng.asserts.Assertion$19.doAssert(Assertion.java:275)
	at org.testng.asserts.Assertion.executeAssert(Assertion.java:30)
	at org.testng.asserts.Assertion.doAssert(Assertion.java:15)
	at org.testng.asserts.Assertion.assertEquals(Assertion.java:272)
	at NotPartOfFramework.LoginListenersTests_UsingSuiteFile.Login_TC1(LoginListenersTests_UsingSuiteFile.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login_TC1 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:NotPartOfFramework.LoginListenersTests_UsingSuiteFile@64bf3bbf]" name="afterMethod" is-config="true" duration-ms="2394" started-at="2019-07-14T18:53:20Z" finished-at="2019-07-14T18:53:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- NotPartOfFramework.LoginListenersTests_UsingSuiteFile -->
    </test> <!-- newCustomer -->
  </suite> <!-- Suite1 -->
</testng-results>
